src/Files.App/App.xaml.cs

```diff
@@ -117,14 +117,14 @@ private static Task StartAppCenter()
 			return Task.CompletedTask;
 		}
 
-		private static async Task InitializeAppComponentsAsync()
+		private static Task InitializeAppComponentsAsync()
 		{
 			var userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
 			var addItemService = Ioc.Default.GetRequiredService<IAddItemService>();
 			var generalSettingsService = userSettingsService.GeneralSettingsService;
 
 			// Start off a list of tasks we need to run before we can continue startup
-			await Task.Run(async () =>
+			return Task.Run(async () =>
```

Comment:
I don't think this behavior is parity with the existing one. I remembered there used to be a crash related this kind of change (the Update process must happen in the main thread), so can you revert the change here?
We may revisit here later.

Reply:
Just reverted the changes here.